src/network/taraxa_capability.cpp

```diff
@@ -1461,15 +1522,20 @@ void TaraxaCapability::onNewPbftVote(taraxa::Vote const &vote) {
       }
     }
   }
-
-  for (auto const &peer : peers_to_send) {
-    sendPbftVote(peer, vote);
+  for (auto const &peerID : peers_to_send) {
+    sendPbftVote(peerID, vote);
   }
 }
 
-void TaraxaCapability::sendPbftVote(NodeID const &_id, taraxa::Vote const &vote) {
-  LOG(log_dg_vote_prp_) << "sendPbftVote " << vote.getHash() << " to " << _id;
-  sealAndSend(_id, PbftVotePacket, RLPStream(1) << vote.rlp(true));
+void TaraxaCapability::sendPbftVote(NodeID const &peerID, taraxa::Vote const &vote) {
+  LOG(log_dg_vote_prp_) << "sendPbftVote " << vote.getHash() << " to " << peerID;
+  auto peer = getPeer(peerID);
+  if (!peer->syncing_ && peer->readyToReceivePackets()) {
```

Comment:
Since getPeer may return nullptr, I think that may cause core dump
Could change:
if (peer && !peer->syncing_ && peer->readyToReceivePackets()) {